### ISSUE TITLE
i18n revision due

### ISSUE THREAD

**Issue by TEITechnicalCouncil** [TEIC/TEI#960]
In the last couple of years there have been a number of small revisions to wording of &lt;desc&gt; or &lt;gloss&gt;, and additions of new elements, which means  that the translations are slightly out of sync. It is time to return to the translators and give them a (hopefully small..) list of things to check.

Original comment by: @sebastianrahtz

**Comment by martindholmes** [comment]
We've made numerous failed attempts to get various groups to engage with translating specific languages (German, Italian, Hebrew) and none of them have come to anything. I'm transitioning off Council, so I'm going to set this ticket to Amber now; I don't believe there's a way to refresh the translations without grant money, and there's no-one I could apply to for it.

**Comment by sydb** [comment]
@peterstadler , @martinascholger , @smjwsk to try organizing a German-language “translat-a-thon” around the Vienna conference 2016-09.

**Comment by sydb** [comment]
Note: this ticket is GREEN for this particular attempt (the Vienna translate-a-thon), but otherwise is RED, as we don’t have a plan for further work.

**Comment by martindholmes** [comment]
What happened in Vienna? Did the translate-a-thon go ahead?

**Comment by peterstadler** [comment]
Yes, the translatathon went pretty well :) The report is forthcoming …

**Comment by martindholmes** [comment]
@peterstadler Did you use the spreadsheet approach? If so, I think the P5/Utilities/mergeLang.xsl stylesheet will merge the results into the P5 source, but we'll need to check for anything that might need updating as a result of Pure ODD (hopefully nothing), and it's been such a long time since this has been used we should do some cautious tests before letting it loose on the whole repo.

Do you think you should have this ticket? I don't think I'm the right person for it.

**Comment by peterstadler** [comment]
Yes, we used a spreadsheet but added the changes to the spec files by hand. This has already happened, see e.g. f47482267cfde14190f2158020a4cb2224e9dc05 

Don't think I'd be a better shepherd of this ticket, at present :(

**Comment by martindholmes** [comment]
Some more progress with this is going to be happening in advance of the conference in Tokyo. The East Asian/Japanese SIG is going to be translating some of the TEI materials on the Dariah site into Japanese, and will then move on to updating the Guidelines translations. I've removed the Blocked status.

**Comment by martindholmes** [comment]
Adding Martina to this ticket because it really doesn't belong with me; I can't lead any particular translation effort, and I'm not on Council either.

**Comment by martindholmes** [comment]
Quick update: a team is working on the Japanese translations using a spreadsheet, and I will be doing the integration of the results back into the specifications over the next few months.

**Comment by martindholmes** [comment]
Another update: still working on integrating the Japanese translations, which is not a trivial task.

**Comment by martindholmes** [comment]
@TEITechnicalCouncil I believe we should close this ticket; it's a bit too general to figure out what action should be taken. I think it's more of a working-group thing. @martinascholger @hcayless Isn't your i18n working group doing this work already?
